bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/Settings.java

```diff
@@ -71,8 +71,7 @@ public Map<String, Object> get() throws LicensingException {
 	}
 
 	private Stream<Path> settingFilesIn(Path path) throws IOException {
-		return Files.walk(path) //
-				.filter(Files::isRegularFile) //
+		return FileCollection.filesIn(path) //
```

Comment:
There is no reason for  static appeal. 
`FileCollection` is aimed to server the only purpose, and it fits here as it is.
Construct `FileCollection` with desired file extension and get its content.

Reply:
Reverted that part as well.